### ISSUE TITLE
Language Service crashes on startup (error code 3221225781) - Windows 10 missing dependencies?

### ISSUE THREAD

**Issue by owennewo** [arduino/arduino-pro-ide#210]
**TLDR**
**Workaround:** Manually installation of `Visual Studio 2015 redistributable` provides the missing `api-ms-win-crt-locale-l1-1-0.dll` which is required by the `language service`.

**Describe the bug**
Newly installed arduino pro 0.0.4 language server crashes (see screenshot and console output below).

Googling error code 3221225781 suggests missing dependencies.  This is a relative clean windows 10 installation and I have no Visual Studio redistributables, are these required? Which version?

**To Reproduce**
1. Start Arduino Pro 0.0.4  
2. An empty starter sketch with setup() / loop() is automatically created
3. The following error message is shown in a popup (bottom left)
```Failed to start Arduino language server: Error: Process exited with exit code: 3221225781```

**Expected behavior**
Language service should start, I'm expecting intellisense / completion to work but ctrl space shows "No suggestions".

**Screenshots**
<img width="956" alt="arduino_language_service_crash" src="https://user-images.githubusercontent.com/15650427/74587811-1701e680-4fef-11ea-9a47-480e0cc466f8.png">

**Desktop (please complete the following information):**
 - OS: Windows 10 (Build 19041 slow ring)
 - Version 0.0.4 alpha

**Additional context**
```daemon INFO Updating libraries index
root ERROR Unexpected exit code '3221225781' when executing "C:\arduino\resources\app\node_modules\arduino-ide-extension\build\clangd.exe" with args: ["--versn"].
languages ERROR Error occurred while starting language contribution. /services/languages/ino/:id. Error: Process exited with exit code: 3221225781
    at ChildProcess.<anonymous> (C:\arduino\resources\app\node_modules\arduino-ide-extension\lib\node\exec-util.js:124:24)
    at ChildProcess.emit (events.js:182:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)```

**Comment by owennewo** [comment]
More info, from git bash I ran the following:
```
$ cd /c/arduino/resources/app/node_modules/arduino-ide-extension/build/
$ ./clangd.exe --version
C:/arduino/resources/app/node_modules/arduino-ide-extension/build/clangd.exe: 
error while loading shared libraries: api-ms-win-crt-locale-l1-1-0.dll: 
cannot open shared object file: No such file or directory```

**Comment by owennewo** [comment]
OK - I've fixed this issue by finding the missing DLL
I've installed Visual Studio 2015 redistributable from the microsoft link below and language service within Arduino IDE Pro is now working
https://www.microsoft.com/en-ca/download/confirmation.aspx?id=48145

<img width="636" alt="arduino_fixed" src="https://user-images.githubusercontent.com/15650427/74588171-13239380-4ff2-11ea-9f96-9c5a739f2139.png">

Perhaps docs and/or install for windows needs to be updated?

**Comment by spoenemann** [comment]
I added the missing DLL. Please reopen if it occurs again.
